pkg/util/file_indexer.go

```diff
@@ -26,14 +27,10 @@ func read(filePath string) (map[string]FileData, error) {
 		return map[string]FileData{}, nil
 	}
 
-	jsonFileRead, err := os.Open(filePath)
-	// if we os.Open returns an error then handle it
+	byteValue, err := ioutil.ReadFile(filePath)
```

Comment:
please add some comments explaining the benefit of this approach as compared to the earlier one

Reply:
This is just DRYing the code

Reply:
But some comments will benefit in the future

Reply:
there is nothing different or better about this approach other than that its less code then before :sweat_smile:

Reply:
and at other places like this https://github.com/openshift/odo/pull/1982#discussion_r310565470. I have provided the reasoning